source/distributed/tria.cc

```diff
@@ -4160,6 +4163,12 @@ namespace parallel
 
       // update how many cells, edges, etc, we store locally
       this->update_number_cache();
+
+      // signal that repartitioning is finished,
+      // this is triggered before update_periodic_face_map
+      // to be consistent with the serial triangulation class
+      this->signals.post_distributed_repartition();
+
       this->update_periodic_face_map();
```

Comment:
I think it might be nice to call the signal after that last function, so that the slots called by the signal can rely on the triangulation to be in a consistent state.

Reply:
I checked what we did in `execute_coarsening_and_refinement()` and placed the signal call for `repartition()` accordingly.
https://github.com/dealii/dealii/blob/97fd41ea479749a3ed959f83ea9093d279a273fb/source/distributed/tria.cc#L4065

I would prefer if both `execute_coarsening_and_refinement()` and `repartition()` would call their signals in a similar position. I'll move both signal calls after the last function and see how testers react.

Reply:
Good plan. Let me know when you have the results. Or leave it as is for the moment and do it in a separate follow-up.